routes/index.js

```diff
@@ -1,7 +1,30 @@
 const express = require('express');
 const router = express.Router();
+const { Movie } = require('../models/Movie.model'); // Copy - pasted // not understood yet
 
 /* GET home page */
 router.get('/', (req, res, next) => res.render('index'));
 
+/* GET movies page */
+router.get('/movies', async (req, res, next) => {
+  Movie.find({}, 'title image')
```

Comment:
this is weird -> on one hand you use async in the handler function, but on the other hand you use a promise. 
The async part is only necessary if you use await... if you use promises instead - i.e. don't have await, there is no need to use async either. 

in case you want to refactor this to not use promises, but async and await instead - this is one way it could look like: 

```js
const data = await Movie.find({}, 'title image').catch(err => console.error(err))
res.render('movies', { Movies: data })
```